sdk/cosmos/microsoft-azure-cosmos-examples/pom.xml

```diff
@@ -6,14 +6,14 @@ Licensed under the MIT License.
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <groupId>com.microsoft.azure</groupId>
+    <groupId>com.azure</groupId>
     <artifactId>azure-cosmos-parent</artifactId>
-    <version>3.3.0</version>
+    <version>4.0.0-preview.4</version>
   </parent>
 
-  <groupId>com.microsoft.azure</groupId>
+  <groupId>com.azure</groupId>
   <artifactId>azure-cosmos-examples</artifactId>
-  <version>3.3.0</version>
+  <version>4.0.0-preview.4</version>
```

Comment:
What I encourage client libraries to do is have a /src/samples directory. This may or may not be a good candidate module for that (in the fullness of time).

Reply:
#5626

Reply:
Noted. Will get back on this